.github/scripts/python.sh

```diff
@@ -65,14 +65,13 @@ function build()
   # Set to 2 cores so that Actions does not error out during resource provisioning.
   cmake --build build -j2
 
-  $PYTHON -m pip install --user build/python
+  cmake --build build --target python-install
 }
 
 function test()
 {
-  cd $GITHUB_WORKSPACE/python/gtsam/tests
-  $PYTHON -m unittest discover -v
-  cd $GITHUB_WORKSPACE
```

Comment:
Please undo these changes. One reason we keep them here is so that others can copy-paste them when trying to run tests, and these changes are tangential to the PR.

Reply:
I will revert. 
people also can copy the cmake commands. It will work on local the same.

Reply:
Also people cannot copy past these line with GITHUB_WORKSPACE environment variable, they need to modify it.
On my lines, they don't need to change anything, plus it bonus that it using the python that was discovered in cmake, ratter than the default python that they have, so in some cases it not the same.

Reply:
When I say copy-paste, I mean with modifications. The original code was also meant to be instructional since people are familiar with how to run Python's unittest module and may not have the same experience with CMake.

Again, if you wish to receive reviews from me, please accept my comments. I only request changes when needed since we have put considerable thought into these things in the past.

Reply:
I already undo these changes.

Reply:
This should be outdated.